.travis.yml

```diff
@@ -28,7 +28,7 @@ addons:
 
 before_install:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update           ; fi
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install qt4      ; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then sh mac-build.sh       ; fi
```

Comment:
I tried to tap another repository to brew and install qt4, but i can't found any stable source.
i also tried this one  `brew install cartr/qt4/qt` and that wasn't successful too :(